src/router/routes.js

```diff
@@ -37,6 +37,10 @@ const routes = [
         path: 'profile',
         component: () => import('pages/ProfilePage.vue')
       },
+      {
+        path: 'test',
+        component: () => import('pages/TestPage.vue')
+      },
```

Comment:
Is the test page necessary in a development environment?

Reply:
No I should remove it

Reply:
Ok, when you commit it, you can push "Resolve conversation" button.